src/lib/metrics/MetricReport.ts

```diff
@@ -0,0 +1,51 @@
+import { SessionManager } from '../session/SessionManager'
+import * as pkg from '../../../package.json'
+
+export interface Metric {
+  command: string
+  [metricName: string]: number | string
```

Comment:
Can a `[metricName]` have a string value?

Reply:
Not for now, but maybe in the future (depending of what metrics we want).